architecture-decisions/0003-preservation.md

```diff
@@ -0,0 +1,67 @@
+# 3. Preservation
+
+Date: 2019-04-02
+
+## Status
+
+Accepted
+
+## Context
+
+We have agreed with our stakeholders that we will preserve our digital objects by packaging them in
+BagIt bags, and storing those bags in cloud storage.  However, there are wide variety of options for
+creating bags, how objects are serialized within the bags, and how they are linked to the digital
+objects.
+
+## Decisions
+
+1. Creating bags
+   1. We will periodically scan for objects which meet the following criteria:
+      1. Status is complete
+      2. Not bagged more recently than their `updated_at` timestamp
+      3. Has a `cloud` preservation policy assigned
+   1. In order to avoid redundant processing when there are multiple updates in a short period of time,
```

Comment:
We do still have a ticket for doing this on create. Remove that ticket and roll everything into this one process?

Reply:
Yes, I think that's where we wound up in our discussion yesterday.